src/main.rs

```diff
@@ -59,26 +59,6 @@ fn exec() -> anyhow::Result<()> {
     let raw_config = std::fs::read(CONFIG_FILE)?;
     let mut config: template::SiteInfo = toml::from_slice(&raw_config)?;
 
-    let client_gzip_support = match std::env::var("HTTP_ACCEPT_ENCODING") {
-        Ok(encoding) => {
-            let mut found = false;
-            for en in encoding.split(",") {
-                if en.trim() == "gzip" {
-                    found = true;
-                    break;
-                }
-            }
-            found
-        }
-        _ => false,
-    };
-
-    //if gzip encoding enabled
-    let gzip_encoding = match config.content_encoding.as_ref() {
```

Comment:
we should probably remove content_encoding from SiteInfo?

```

#[derive(Serialize, Deserialize)]
pub struct SiteInfo {
    pub title: String,
    pub logo: Option<String>,
    pub base_url: Option<String>,
    pub content_encoding: Option<String>,
    pub about: Option<String>,
    pub extra: BTreeMap<String, String>,
}
```

Reply:
Good catch! Thanks!